web/src/stream_edit.js

```diff
@@ -83,17 +83,7 @@ function get_stream_id(target) {
 
 function get_sub_for_target(target) {
     const stream_id = get_stream_id(target);
-    if (!stream_id) {
-        blueslip.error("Cannot find stream id for target");
-        return undefined;
-    }
-
-    const sub = sub_store.get(stream_id);
-    if (!sub) {
-        blueslip.error("get_sub_for_target() failed id lookup", {stream_id});
-        return undefined;
-    }
-    return sub;
+    return sub_store.get(stream_id);
```

Comment:
This might be OK if the final version does assertions, which I guess can be checked.

Reply:
Yes, an assertion is added here in the last commit.